docs/Filament.md.html

```diff
@@ -1902,6 +1902,94 @@
 
 ##### Simplifying the BRDF integration #####
 
+Since there is no closed-form or an easy way to compute the $\Lout$ integral, we use a simplified
+equation instead, $\hat{I}$, whereby we assume that $v = n$, that is the view direction $v$ is always
+equal to the surface normal $n$. Clearly, this assumption will break all view-dependant effects of 
+the convolution, such as the increased blur in reflections closer to the viewer 
+(a.k.a. stretchy reflections). 
+
+Such a simplification would also have a severe impact on constant environments, such as the white 
+furnace, because it would affect the magnitude of the the constant (i.e. DC) term of the result. We
+can at least correct for that by using a scale factor, $K$, in our simplified integral, which,
+when chosen properly will make sure the average irradiance stay correct.
```

Comment:
comma after "when chosen properly"
or remove commas and move when chosen properly to the end